modules/tinymce/changelog.txt

```diff
@@ -1,15 +1,15 @@
 Version 5.7.0 (TBD)
     Added IPv6 address support to the URI API. Patch contributed by dev7355608 #GH-4409
-    Added new data to the `TableModified` event to describe whether the table's `structure`, `style`, or both were modified #TINY-6643
-    Added a new locked column feature to tables that allows column operations to be prevented on certain columns #TINY-6765
+    Added new data to the `TableModified` event to describe whether the table `structure`, `style`, or both was modified #TINY-6643
```

Comment:
Are you sure about this, reading it aloud sounds really strange?

Reply:
```suggestion
    Added two optional return parameters to the `TableModified` event indicating if the `structure` or `style` was modified #TINY-6643
```
?

Reply:
`Optional` is incorrect as they are always provided. They also aren't `return parameters`, they are properties in the event object/type.

Reply:
Also, it's not exclusively either-or, it can be any of the following:
- `{ style: false, structure: true }` (structure change)
- `{ style: true, structure: false }` (style change)
- `{ style: true, structure: true }` (both structure and style change)

Reply:
```suggestion
    Added new properties to the `TableModified` event to describe whether the table `structure`, `style`, or both were modified #TINY-6643
```

Reply:
```suggestion
    Added new properties to the `TableModified` event to indicate if the table `structure` or `style` were modified #TINY-6643
```
removed whether so it doesn't imply it's mutually exclusive, so that we maybe don't need the `or both`? wondering if the backticks are needed...

Reply:
```suggestion
    Added new `structure` and `style` properties to the `TableModified` event to describe how the table was modified #TINY-6643
```
or something maybe? rearranging the sentence...

Reply:
LGTM :+1:

To clarify, I was originally asking about changing `were` to `was` as that seemed wrong. So this really just spiralled out of control sorry :disappointed:

Reply:
it read weird to me, I only figured out why when I saw your comments